packages/esm-appointments-app/src/form/appointments-form.workspace.tsx

```diff
@@ -426,7 +432,9 @@ const AppointmentsForm: React.FC<AppointmentsFormProps & DefaultWorkspaceProps>
     const hours = (hoursAndMinutes[0] % 12) + (timeFormat === 'PM' ? 12 : 0);
     const minutes = hoursAndMinutes[1];
     const startDatetime = startDate.setHours(hours, minutes);
-    const endDatetime = dayjs(startDatetime).add(duration, 'minutes').toDate();
+    const endDatetime = allowAllDayAppointments
```

Comment:
```suggestion
    const endDatetime = isAllDayAppointment
```

Should this use the `isAllDayAppointment` form field destructured from `data` on lines 416-428 instead of the original config property?